cluster/watchdog.go

```diff
@@ -121,6 +121,9 @@ func (w *Watchdog) rescheduleContainers(e *Engine) {
 			}
 		}
 
+		// Clear out the network configs, since we cannot create a
+		// container with multiple networks.
+		c.Config.NetworkingConfig.EndpointsConfig = nil
```

Comment:
It appears the only other type of scope is `local`:

https://github.com/docker/docker/blob/5497f4758d14f6f65c722838d342f634e353b08e/api/server/router/network/network_routes.go#L182
https://github.com/docker/libnetwork/blob/82905ed5312c9b3403e5266aab5c2b2dd346ce5a/datastore/datastore.go#L113